packages/react/src/molecules/pickers/select/base.tsx

```diff
@@ -114,6 +115,10 @@ const PickerSelectBase = React.forwardRef(
             <LabelBase {...props.labelBaseProps} />
           )}
 
+          {props.rightTextProps !== undefined && (
+            <LabelBase {...props.rightTextProps} />
```

Comment:
@ShreyDhyani @Subbu-betalectic I don't think should be LabelBase right? This can be any component

Reply:
@Subbu-betalectic You need to just add something like 

```

<span className="cursor-pointer" onClick={() => setSelected([])}>
              <props.customRightComponent />
            </span>
```